build_container/build_container_centos.sh

```diff
@@ -49,3 +49,5 @@ if [[ $(uname -m) == "aarch64" ]] && grep -q -e rhel /etc/*-release ; then
 fi
 
 source ./build_container_common.sh
+
+yum clean all
```

Comment:
This was added to ensure the images get rebuild and re-pushed, otherwise since the CI build that failed on master was cancelled after the docker hub image was pushed but before the GCR image, we are in a weird state

Reply:
Removed since vs build tools installs are failing across the board (locally etc.), their server is failing to serve a file that is part of the installation